### DISCUSSION TITLE
Redis Shake 错误总结

### DISCUSSION THREAD

**Discussion by giscus** [Lee-CQ/hugo-blog#3] [Announcements]
# Redis Shake 错误总结

在使用redis Shake 中遇到的一些奇奇怪怪的问题。

https://jubilant-giggle-65xjq99wjpfwvv-1313.app.github.dev/p/redis-shake-%E9%94%99%E8%AF%AF%E6%80%BB%E7%BB%93/

**Comment by lee-cq** [comment]
Redis Shake 不太适用于持久同步
